_dev/src/utils/Sentry.ts

```diff
@@ -8,6 +8,9 @@ if (store.state.app.psxMktgWithGoogleOnProductionEnvironment) {
     Vue,
     // @ts-ignore
     dsn: global.psxMktgWithGoogleDsnSentry,
+    allowUrls: [
```

Comment:
looks like it is `whitelistUrls` in the [docs](https://docs.sentry.io/clients/javascript/config/) though

Reply:
whitelistUrls is deprecated

Reply:
Where did you find this information?

Reply:
https://docs.sentry.io/platforms/javascript/guides/vue/configuration/filtering/